crates/cast/bin/opts.rs

```diff
@@ -859,6 +859,10 @@ pub enum Subcommands {
         #[clap(value_name = "BYTES")]
         bytes: Option<String>,
     },
+
+    /// Decodes a raw signed EIP 2718 typed transaction
+    #[clap(visible_alias = "d")]
```

Comment:
I think --decode is not meaningful enough, because there are already a lot of "decode" functions, maybe `--decode-transaction`?